app/renderer/components/main/main.js

```diff
@@ -275,16 +257,23 @@ class Main extends React.Component {
 
     ipc.on('scroll-touch-edge', () => {
       if (trackingFingers) {
-        if (!isSwipeOnRightEdge && deltaX > 0) {
-          isSwipeOnRightEdge = true
-          isSwipeOnLeftEdge = false
-          time = 0
-          deltaX = 0
-        } else if (!isSwipeOnLeftEdge && deltaX < 0) {
-          isSwipeOnLeftEdge = true
-          isSwipeOnRightEdge = false
-          time = 0
-          deltaX = 0
+        const distanceThresholdX = getSetting(settings.SWIPE_NAV_DISTANCE)
+        const percent = Math.abs(deltaX) / distanceThresholdX
+        time = (new Date()).getTime() - startTime
+        if (deltaX > 0) {
+          if (percent > 1) {
+            appActions.swipedRight(1)
+          } else {
+            appActions.swipedRight(percent)
+          }
+          deltaX += 1
```

Comment:
what's the intended purpose of this line and line 276? that seems to cause the issue i mentioned in the discussion

Reply:
again, that is intentional for `features::kTouchpadAndWheelScrollLatching` enabled in c69 and master does't have the compatible version of muon now.
could you try 0.23.x-c69 branch first?

Reply:
did you delete `node_modules/electron-prebuilt` and redo a npm install?
If you didn't, that means you don't have the correct muon version.

Reply:
you can also try https://github.com/brave/browser-laptop/releases/tag/v0.23.202beta which is just built from that branch and contains this fix

Reply:
yes i rm rf'd node modules on the `0.23.x-c69` branch and the issue still occurs. Also the issue i mentioned still occurs in https://github.com/brave/browser-laptop/releases/tag/v0.23.202beta

would you like me to try to fix?

Reply:
I believe the problem you talked about is no way to cancel the navigation once you reach 100%. you can try to fix that. But I don't see so called incredibly high sensitivity

Reply:
I'm going to emit more attributes from muon for this event so I can update the precise units

Reply:
you can try 0.23.x-c69 again now. new muon build is ready